reference/recency-surveillance.v1.json

```diff
@@ -177,25 +191,30 @@
                         ]
                       },
                       "number_of_sexual_partners_last_12_months": {
+                        "description": "Number of sexual partners twelve months ago", 
                         "type": "integer",
                         "default": "integer"
                       },
                       "number_of_tests_last_12_months": {
+                        "description": "Number of times tested twelve months ago",
                         "type": "integer",
                         "default": "integer"
                       },
                       "last_hiv_test_result": {
+                        "description": "Test results the last time tested",
                         "type": "string",
                         "enum": [
                           "HIV+",
                           "HIV-"
                         ]
                       },
                       "last_hiv_test_date": {
+                        "description": "Must be YYYY-MM-DD format",
```

Comment:
Its small yyyy capital MM and small dd. The casing is important. Same for the other date fields as well.

Reply:
@mugumedavid Done